components/use-cases-card/use-cases-card.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
```

Comment:
I'm seeing some weirdness with the medium-size breakpoint. There's a blank space on the right side. I think this is from the issue in the `Why Kedro` section I mentioned above. 

<img width="930" alt="image" src="https://user-images.githubusercontent.com/16869061/157003453-8dda35a4-2ed6-4f00-9bfb-c30a919e79d8.png">

And in general, it seems the spacing needs to be tightened up a bit on the desktop size and increased a bit on the mobile size. See in red where it needs to be tightened on desktop: 

<img width="1287" alt="Screen Shot 2022-03-07 at 11 25 25" src="https://user-images.githubusercontent.com/16869061/157004133-88ae1368-9a56-480c-ad23-559db1c311f1.png">

Reply:
Thanks for the style comments - I've resolved all of these 'weirdness' in my latest commits - please also see the screenshots I've posted on the description section above with the new improved layout